soak/README.md

```diff
@@ -0,0 +1,202 @@
+## INTRODUCTION
+
+ACK team provides the soak testing framework for service teams where service teams can execute long running
+soak tests for testing the performance and stability of the service controllers.
+
+Under the hood, these soak tests consist of multiple end to end test runs for the service controller. End to end tests are present in service-controller github repository.
+i.e. "https://github.com/aws-controllers-k8s/<service-name>-controller".
+
+If you wish to run soak tests for your service controller locally, see the "HOW TO RUN SOAK TEST LOCALLY" section below.
+
+## RUNNING SOAK TESTS USING ACK PROW JOBS
+
+TBD
+
+
+## HOW TO RUN SOAK TEST LOCALLY
+
+TODO
+```
+* Convert following 9 steps into a single executable script with default ack soak-test params. 
+* Service teams will have the freedom to set parameters and customizing their soak-test runs.
+```
+
+### Prerequisites
+* A tool for building OCI images (docker, buildah etc..)
+* A kubernetes cluster to run soak tests
+* helm 
+* kubectl
+
+
+### STEP 1 (Install the ACK service controller using helm)
+
+In this step we install ack service controller which will be soak tested using the soak-test runner created in step 4.
+
+You can install an ACK controller following the documentation [here](https://aws-controllers-k8s.github.io/community/user-docs/install/)
+but following commands are also TL;DR for the installation documentation.
+
+You can use either (a) or (b) method below.
+
+a) Pull the chart from public repo
```

Comment:
This option should not be valid because soak tests are run before we release into the public chart repo. I'd only recommend (b)

Reply:
>"soak tests are run before we release into the public chart repo"

This will only be true for stable release. For patch releases (semver update), soak tests will run after the release and before marking the release as stable.